### ISSUE TITLE
having trouble reproducing results for esol, freesolv, and bace

### ISSUE THREAD

**Issue by orgw** [learningmatter-mit/matex#1]
Hi, thanks for the code. 

i'm having trouble reproducing results for esol, freesolv, and bace. i'm constantly getting worse results compared to those in your paper (esol(delaney) is getting 0.54 unlike 0.08 in the paper) . maybe the hyperparameters are wrong in train_eval.sh? i've got better results for lipo using the params.

also, is there a standard for when to stop training? it seems that the ood and eval metric seems to fluctuate a lot depending on when you stop.
